specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json

```diff
@@ -0,0 +1,2322 @@
+                  "description": "Brief description of cause of the resource becoming unavailable."
+                }
+              }
+            },
+            "recommendedActions": {
+              "type": "array",
+              "description": "Lists actions the user can take based on the current availabilityState of the resource.",
+              "items": {
+                "$ref": "#/definitions/recommendedAction"
+              },
+              "x-ms-identifiers": [
+                "action"
+              ]
+            },
+            "serviceImpactingEvents": {
+              "type": "array",
+              "description": "Lists the service impacting events that may be affecting the health of the resource.",
+              "items": {
+                "$ref": "#/definitions/serviceImpactingEvent"
+              },
+              "x-ms-identifiers": [
+                "correlationId"
+              ]
+            }
+          }
+        }
+      }
+    },
+    "operationListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/operation"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ],
+          "description": "List of operations available in the resourcehealth resource provider."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "Lists the operations response."
+    },
+    "operation": {
+      "description": "Operation available in the resourcehealth resource provider.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the operation."
+        },
+        "display": {
+          "type": "object",
+          "description": "Properties of the operation.",
+          "properties": {
+            "provider": {
+              "type": "string",
+              "description": "Provider name."
+            },
+            "resource": {
+              "type": "string",
+              "description": "Resource name."
+            },
+            "operation": {
+              "type": "string",
+              "description": "Operation name."
+            },
+            "description": {
+              "type": "string",
+              "description": "Description of the operation."
+            }
+          }
+        }
+      }
+    },
+    "recommendedAction": {
+      "type": "object",
+      "description": "Lists actions the user can take based on the current availabilityState of the resource.",
+      "properties": {
+        "action": {
+          "type": "string",
+          "description": "Recommended action."
+        },
+        "actionUrl": {
+          "type": "string",
+          "description": "Link to the action"
+        },
+        "_ActionUrl.Comment": {
+          "type": "string",
+          "description": "the comment for the Action"
+        },
```

Comment:
Why is it this funny looking property name '_ActionUrl.Comment'?

Reply:
And not e.g. 'actionUrlComment'?

Reply:
apparently this is an old field that is still supported. we recieved the following s360: https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=647F3368-7670-4ADF-9D8D-6A6F02006D3F 

we can't change it without a breaking change